src/components/ProductInfo.vue

```diff
@@ -0,0 +1,392 @@
+<template>
+  <div v-if="!empty && !loading">
+    <div data-test="product-gallery"
+         class="col-md-4 col-md-offset-1 col-sm-6 product-gallery">
+      <ProductGallery v-if="images"
+                      :productImages="images" />
+    </div>
+    <div data-test="product-description"
+         class="col-sm-6 product-description">
+      <div class="row">
+        <div class="col-sm-12">
+            <h1 data-test="pdp-product-title"
+                class="text-uppercase pdp-product-title">
+              {{currentProduct.name}}
+            </h1>
+            <span data-test="quickview-sku"
+                  class="grey-p quickview-sku">
+              {{matchingVariant.sku}}
+            </span>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <!-- {{> catalog/product-rating }} -->
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <p class="pdp-product-description view-details more"
+             data-text-show="$t('description.show')"
+             data-text-hide="$t('description.hide')">
+          </p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div v-if="hasPrice"
+             class="col-sm-12">
+          <p class="product-price">
+            <span v-if="!hasDiscount"
+                  data-test="product-original-price">
+              {{ formatPrice(originalPrice) }}
+            </span>
+            <span v-else>
+              <span data-test="product-discounted-price"
+                    class="discounted-price">
+                {{ formatPrice(originalPrice) }}
+              </span>
+              <span data-test="product-discount-price">
+                {{ formatPrice(discountedPrice) }}
+              </span>
+            </span>
+          </p>
+
+        </div>
+      </div>
+      <!-- {{> catalog/add-to-cart}}
+      {{> catalog/add-to-wishlist-btn}}
+      {{> catalog/reserve-in-store-btn}} -->
+      <button id=""
+              data-test="add-to-cart-button"
+              class="add-to-bag-btn">
+        <img class="bag-thumb"
+             src="../assets/img/hand-bag-2-black.png"
+             alt="$t('catalog.cart.add')">
```

Comment:
This should also be moved into the local `<i18n>`, right? If it's used anywhere else what we can do is create a component for this button and put it there.